pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,181 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Cluster onboarding\n",
+    "This guide explains how you can onboard to one of the Operate First clusters. First chapter lists different environments and clusters where you can onboard, second explains how you can onboard as user and third how you can onboard yourself creating PR.\n",
+    ":::{seealso}\n",
+    "For more information about the Operate First see [Getting started with Operate First](references:operate-first-start)\n",
+    ":::\n",
+    "\n",
+    "## User\n",
+    "\n",
+    "Clusters managed by Operate First are listed [here](references:clusters).\n",
+    "To request access to one of these clusters please file a GitHub issue [here](https://github.com/operate-first/support/issues/new?assignees=first-operator&labels=kind%2Fonboarding%2Carea%2Fcluster&template=onboarding_to_cluster.yaml&title=NEW+PROJECT%3A+%3Cname%3E). \n",
+    "\n",
+    "\n",
+    "## Operation\n",
+    "\n",
+    "This part of tutorial servers as a guide for adding cluster scoped resources of `Namespace` and `Group` kinds. Following steps in this guide should result in a PR against the [operate-first/apps](https://github.com/operate-first/apps) repository.\n",
+    "\n",
+    "### Prerequisites\n",
+    "There are no prerequisites.\n",
+    "\n",
+    "### Recipe\n",
+    "If you want to know more about the overall design please consult the ADR documentation at [operate-first/support](https://github.com/operate-first/blueprint).\n",
+    "In general we store all the cluster-scoped resources in a `cluster-scope` kustomize application within this repository.\n",
+    "#### Getting repository \n",
+    "\n",
+    "Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. During this whole setup, we’ll be working within this repository.\n",
```

Comment:
Show it! 

1. Go to url
2. click fork
3. run following command in here: `git clone` + output